diesel/src/query_builder/select_statement/boxed.rs

```diff
@@ -22,28 +22,28 @@ use sql_types::{BigInt, Bool, NotNull, Nullable};
 
 #[allow(missing_debug_implementations)]
 pub struct BoxedSelectStatement<'a, ST, QS, DB> {
-    select: Box<QueryFragment<DB> + 'a>,
+    select: Box<QueryFragment<DB> + Send + 'a>,
     from: QS,
-    distinct: Box<QueryFragment<DB> + 'a>,
+    distinct: Box<QueryFragment<DB> + Send + 'a>,
     where_clause: BoxedWhereClause<'a, DB>,
-    order: Option<Box<QueryFragment<DB> + 'a>>,
-    limit: Box<QueryFragment<DB> + 'a>,
-    offset: Box<QueryFragment<DB> + 'a>,
-    group_by: Box<QueryFragment<DB> + 'a>,
+    order: Option<Box<QueryFragment<DB> + Send + 'a>>,
+    limit: Box<QueryFragment<DB> + Send + 'a>,
+    offset: Box<QueryFragment<DB> + Send + 'a>,
+    group_by: Box<QueryFragment<DB> + Send + 'a>,
     _marker: PhantomData<ST>,
 }
 
 impl<'a, ST, QS, DB> BoxedSelectStatement<'a, ST, QS, DB> {
     #[allow(clippy::too_many_arguments)]
     pub fn new(
-        select: Box<QueryFragment<DB> + 'a>,
+        select: Box<QueryFragment<DB> + Send + 'a>,
         from: QS,
-        distinct: Box<QueryFragment<DB> + 'a>,
+        distinct: Box<QueryFragment<DB> + Send + 'a>,
         where_clause: BoxedWhereClause<'a, DB>,
-        order: Option<Box<QueryFragment<DB> + 'a>>,
-        limit: Box<QueryFragment<DB> + 'a>,
-        offset: Box<QueryFragment<DB> + 'a>,
-        group_by: Box<QueryFragment<DB> + 'a>,
+        order: Option<Box<QueryFragment<DB> + Send +'a>>,
```

Comment:
According to rustfmt, this should be as follows:
```suggestion
        order: Option<Box<QueryFragment<DB> + Send + 'a>>,
```